client/src/rr/dnssec/keypair.rs

```diff
@@ -463,18 +463,8 @@ impl KeyPair {
             Algorithm::RSASHA1 |
             Algorithm::RSASHA1NSEC3SHA1 |
             Algorithm::RSASHA256 |
-            Algorithm::RSASHA512 => {
-                Err(
-                    DnsSecErrorKind::Message("openssl does not yet support pkcs8").into(),
-                )
-            }
-            #[cfg(feature = "openssl")]
-            Algorithm::ECDSAP256SHA256 => {
-                Err(
-                    DnsSecErrorKind::Message("openssl does not yet support pkcs8").into(),
-                )
-            }
-            #[cfg(feature = "openssl")]
+            Algorithm::RSASHA512 |
+            Algorithm::ECDSAP256SHA256 |
```

Comment:
these are not equivalent given the restriction on the feature... this might be why you were seeing some compilation issues with different features?

Reply:
Why aren't they equivalent? They are all behing `#[cfg(feature = "openssl")]` no?

Reply:
Oh! You're correct. The github diff cutoff the #[cfg...] line just before showing the diff. I didn't realize it was there.